modules/microshift-install-rpms-gitops.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-rpms-for-gitops_{context}"]
+== Installing the GitOps ArgoCD manifests from an RPM package
+
+You can use a lightweight version of the OpenShift GitOps with {microshift-short} to help manage your applications. Install the necessary ArgoCD manifests using an RPM package. This RPM package included the necessary manifests that runs core Argo CD. 
+
+.Prerequisites
+
+* You installed {microshift-short} version 4.14 or higher
+* Additional RAM storage of 250MB recommended 
+
+.Procedure
+
+. Enable the GitOps repository with the subscription manager by running the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos --enable=gitops-1.12-for-rhel-9-<architecture>-rpms
+----
+
+. Install the GitOps packages by running the following command: 
++
+[source,terminal]
+----
+$ sudo dnf install -y microshift-gitops 
+----
+
+. To complete the installation, restart the {microshift-short} service by running the following command: 
```

Comment:
```suggestion
. To deploy Argo CD pods, restart the {microshift-short} service by running the following command: 
```
From my point of view, `microshift-gitops` is already installed in the previous step, but by restarting `microshift` we can start Argo CD pods.

Sometimes we all use `installation` as a general concept that means something is ready to be used, so not sure what's the right wording for this step.